packages/fast-development-site-react/src/components/shell/action-bar.tsx

```diff
@@ -1,24 +1,25 @@
 import * as React from "react";
 import manageJss, { ComponentStyles, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
+import { toPx } from "@microsoft/fast-jss-utilities";
 import { IDevSiteDesignSystem } from "../design-system";
 
 /* tslint:disable-next-line */
 export interface IShellActionBarProps {}
 
 export interface IShellActionBarManagedClasses {
-    shell_actionBar: string;
+    shellActionBar: string;
 }
 
 const style: ComponentStyles<IShellActionBarManagedClasses, IDevSiteDesignSystem> = {
-    shell_actionBar: {
+    shellActionBar: {
```

Comment:
@nicholasrice same question here if you can jump in. This seems to imply that anything that could possibly be a child of another component should follow this convention. That seems wrong as Action Bar in this case is a component itself. If there were nested things within here I could see using camel casing.